.github/workflows/deploy.yml

```diff
@@ -0,0 +1,41 @@
+name: Build and push container images
+
+on:
+  push:
+    branches:
+      - deploy/*
+
+jobs:
+  build-and-push-images:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to the container registry
+        uses: docker/login-action@v1
+        with:
+          registry: gcr.io
+          username: _json_key
+          password: ${{ secrets.GCR_JSON_KEY }}
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build and push radicle-node
+        id: radicle-node
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: radicle-node/Dockerfile
+          push: true
+          tags: gcr.io/radicle-services/radicle-node:latest,gcr.io/radicle-services/radicle-node:${{ github.sha }}
+          cache-from: type=registry,ref=gcr.io/radicle-services/radicle-node:latest
+          cache-to: type=inline
+      - name: Build and push radicle-httpd
+        id: http_api
```

Comment:
Can we change this to `radicle-httpd` as well?

Reply:
Done ✅